crates/ef-testing/src/sequencer/utils.rs

```diff
@@ -0,0 +1,92 @@
+use blockifier::abi::abi_utils::get_storage_var_address as blockifier_get_storage_var_address;
+use reth_primitives::{Bytes, TransactionSigned};
+use reth_rlp::Decodable;
+use revm_primitives::U256;
+use starknet::{
+    core::types::{BroadcastedInvokeTransaction, FieldElement},
+    macros::selector,
+};
+use starknet_api::{
+    core::{ClassHash, ContractAddress},
+    hash::StarkFelt,
+    state::StorageKey,
+};
+
+use super::{constants::KAKAROT_ADDRESS, KakarotSequencer};
+
+pub(crate) fn get_storage_var_address(
```

Comment:
do we really need this wrapper?

Reply:
Just easier for error management for now, allows me to not start to worry about the error I return

Reply:
but you're just wrapping the blockifier with no change?

Reply:
Yeah just wrapping and returning a eyre::Error